types/types.override.pb.go

```diff
@@ -1,7 +1,30 @@
 package types
 
-// XXX_WellKnownType is a hack -- please don't make a pattern out of this!
-// This is used to trick certain toolsets to treat this field like a WKT StringValue.
-// See https://github.com/golang/protobuf/blob/70c277a8a150a8e069492e6600926300405c2884/jsonpb/jsonpb.go#L157
-// and https://github.com/golang/protobuf/blob/70c277a8a150a8e069492e6600926300405c2884/jsonpb/jsonpb.go#L191
-func (*UUIDValue) XXX_WellKnownType() string { return "StringValue" }
+import (
```

Comment:
I feel like you should have your `marshal_tests` here that relate to `UUID` and `JSONValue` instead of having them getting coincidentally tested in the `feature_demo` package. It could happen that somewhere down the line that `feature_demo` package gets removed and then we suddenly lose test coverage on these types.

Reply:
Good call